mm2src/mm2_main/Cargo.toml

```diff
@@ -126,7 +126,7 @@ coins_activation = { path = "../coins_activation", features = ["for-tests"] }
 mm2_test_helpers = { path = "../mm2_test_helpers" }
 mocktopus = "0.8.0"
 testcontainers = "0.15.0"
-web3 = { git = "https://github.com/KomodoPlatform/rust-web3", tag = "v0.19.0", default-features = false, features = ["http"] }
+web3 = { git = "https://github.com/KomodoPlatform/rust-web3", tag = "v0.19.0", default-features = false, features = ["http-tls"] }
```

Comment:
I think one of these https://github.com/KomodoPlatform/rust-web3/blob/01de1d732e61c920cfb2fb1533db7d7110c8a457/Cargo.toml#L78-L79 features adds less dependencies to the dependency stack. Please correct me if I am wrong.

Reply:
Yep `rustls-tls` is a bit smaller, `http-native-tls` doesnt change lock file, seems like its `default-tls` in reqwest.
added here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2100/commits/4478ccb9041a180b3680ae92feda3916fff87769